docs/ECOS.md

```diff
@@ -189,10 +189,13 @@ otherwise dhcp is enabled.
 
 We use several files to run application comes from Docker/OCI manifest:
 
-* `environment` - all environment variables defined in Docker/OCI goes here
+* `environment` - all environment variables defined in Docker/OCI and user data from controller goes here
 * `cmdline` - it is cmd defined to run in Docker/OCI
 * `ug` - file contains user and group to run cmd under
 
+`cmdline` may be overridden with defined `EVE_ECO_ENTRYPOINT` (to override or define entrypoint part)
+and `EVE_ECO_CMD` (to override or define cmd part) environment variables.
```

Comment:
Can we make the description and ECO* names more clear?
It seems like EVE_ECO_ENTRYPOINT can include arguments (cmd + args) and EVE_ECO_CMD is just appended to EVE_ECO_ENTRYPOINT.
That isn't clear from the names and desctiption.

Reply:
After some thought, I decided that one parameter (EVE_ECO_CMD) is enough. It overrides cmdline without any speculations on entrypoint/cmd from manifest.